pkg/api/kubegraph/nodes/types.go

```diff
@@ -38,13 +40,17 @@ func (n ServiceNode) String() string {
 }
 
 func (n ServiceNode) ResourceString() string {
-	return "svc/" + n.Name
+	return "service/" + n.Name
```

Comment:
I didn't overthink it, the first default output we had when creating or manipulating a resource would show the original name and not the abbreviation. Reverted though, don't feel strongly.